src/sync-pending-events.ts

```diff
@@ -138,7 +129,7 @@ export class EventTracker {
     try {
       let pastEvents = await this.contract.getPastEventsFromSteped(lastCheckedBlockNumber + 1, latestBlockNumber, this.eventsSyncStep);
```

Comment:
lastCheckedBlockNumber is get from getLastMonitorBlock, why call it lasted checked block number if you get from a function called getLastMonitorBlock?